src/main/java/dp/xlsx/DatasetFormatter.java

```diff
@@ -84,20 +78,32 @@ void format() {
             sheet.setColumnWidth(columnWidth.getKey(),
                     (columnWidth.getValue() + DIMENSION_WIDTH_PADDING_CHARS) * EXCEL_CHARS_TO_WIDTH_FACTOR);
         }
+
+        if (datasetMetadata.getUserNotes() == null) return;
```

Comment:
picky I know but can you put the brackets in for this `if ` thanks.

Reply:
I inverted the if statement and moved for loop into it. 👍